fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx

```diff
@@ -0,0 +1,356 @@
+  const route = useRoute<NavigationProps['route']>();
+  const { eventId: rollCallId } = route.params;
+
+  const selectRollCall = useMemo(() => makeRollCallSelector(rollCallId), [rollCallId]);
+  const rollCall = useSelector(selectRollCall);
+
+  if (!rollCall) {
+    throw new Error(`Could not find a roll call with id ${rollCallId}`);
+  }
+
+  return <Text style={Typography.topNavigationHeading}>{rollCall.name}</Text>;
+};
+
+export const ViewSinglRollCallScreenRightHeader = () => {
+  const navigation = useNavigation<NavigationProps['navigation']>();
+  const route = useRoute<NavigationProps['route']>();
+  const { eventId: rollCallId, isOrganizer, attendeePopTokens } = route.params;
+
+  const toast = useToast();
+
+  const showActionSheet = useActionSheet();
+
+  const selectRollCall = useMemo(() => makeRollCallSelector(rollCallId), [rollCallId]);
+  const rollCall = useSelector(selectRollCall);
+  if (!rollCall) {
+    throw new Error(`Could not find a roll call with id ${rollCallId}`);
+  }
+
+  const laoId = RollCallHooks.useCurrentLaoId();
+  if (!laoId) {
+    throw new Error('no LAO is currently active');
+  }
+
+  // don't show a button for non-organizers
+  if (!isOrganizer) {
+    return null;
+  }
+
+  // Once the roll call is opened the first time, idAlias is defined, and needed for closing/reopening the roll call
+  const eventHasBeenOpened = rollCall.idAlias !== undefined;
+
+  const makeToastErr = (error: string) => {
+    toast.show(error, {
+      type: 'danger',
+      placement: 'bottom',
+      duration: FOUR_SECONDS,
+    });
+  };
+
+  const onOpenRollCall = () => {
+    requestOpenRollCall(laoId, rollCall.id).catch((e) => {
+      makeToastErr('Unable to send roll call open request');
+      console.debug('Unable to send Roll call open request', e);
+    });
+  };
+
+  const onReopenRollCall = () => {
+    if (eventHasBeenOpened) {
+      requestReopenRollCall(laoId, rollCall.idAlias).catch((e) => {
+        makeToastErr('Unable to send Roll call re-open request');
+        console.debug('Unable to send Roll call re-open request', e);
+      });
+    } else {
+      makeToastErr('Unable to send roll call re-open request, the event does not have an idAlias');
+      console.debug('Unable to send roll call re-open request, the event does not have an idAlias');
+    }
+  };
+
+  const onScanAttendees = () => {
+    console.log('onScanAttendees');
+    if (eventHasBeenOpened) {
+      console.log('eventHasBeenOpened');
+      navigation.navigate(STRINGS.navigation_app_lao, {
+        screen: STRINGS.navigation_lao_events,
+        params: {
+          screen: STRINGS.navigation_lao_events_open_roll_call,
+          params: { rollCallId: rollCall.id.toString() },
+        },
+      });
+    } else {
+      makeToastErr('Unable to scan attendees, the event does not have an idAlias');
+      console.debug('Unable to scan attendees, the event does not have an idAlias');
+    }
```

Comment:
Since these strings will be displayed in the UI (toast message), I guess you can add them in `string.ts`.